packages/integrations/vercel/test/isr.test.js

```diff
@@ -53,6 +49,10 @@ describe('ISR', () => {
 				src: '^\\/two\\/?$',
 				dest: '/_isr?x_astro_path=$0',
 			},
+			{
+				src: '^\\/_image$',
+				dest: '_render',
+			},
```

Comment:
The order of the routes here changed, because the routing priority is supposed to be different for prerendered routes, but this was not properly taken into account before since during the creation of the manifest, we didn't know if a page was prerendered or not.